pkg/api/controller.go

```diff
@@ -25,6 +25,10 @@ import (
 	"zotregistry.io/zot/pkg/log"
 	"zotregistry.io/zot/pkg/scheduler"
 	"zotregistry.io/zot/pkg/storage"
+	"zotregistry.io/zot/pkg/storage/cache"
+	"zotregistry.io/zot/pkg/storage/cache/boltdb"
+	cachedbfactory "zotregistry.io/zot/pkg/storage/cache/factory"
```

Comment:
s/cachedb/cache/g or "cachebackend"
Let's not indicate that the cache backend is always only a db.

Reply:
I would say this is not really a cache, it is a DB. We need to persist data there - for local this cache may be optional, but for S3 it is mandatory to persist as we can't recover the content otherwise.

Reply:
Moved factory to backend, renamed cachedbfactory to cacheBackend